boot.py

```diff
@@ -217,6 +217,20 @@ def on_new_window_async(self, w: sublime.Window) -> None:
     def on_pre_close_window(self, w: sublime.Window) -> None:
         windows.discard(w)
 
+    def on_pre_move(self, view: sublime.View) -> None:
+        listener = windows.listener_for_view(view)
+        if not listener:
+            return
+        sublime.set_timeout_async(listener.on_pre_move_window_async)
+
+    def on_post_move(self, view: sublime.View) -> None:
+        # Note: I'm afraid to use on_post_move_async because I don't know for sure whether that will run *after* the
+        # sublime.set_timeout_async in on_pre_move
+        listener = windows.listener_for_view(view)
+        if not listener:
+            return
+        sublime.set_timeout_async(listener.on_post_move_window_async)
```

Comment:
Those are also called when dragging tabs within the same window (even the same group).

Not sure if optimization that checks if the window was changed is worth it...
Or maybe it should be reported as a bug that it fires even within the same window?

Reply:
We're checking if the window IDs are the same and bail out if so.